test/Debugger/JsDiagGetStackProperties.js.dbg.baseline

```diff
@@ -0,0 +1,107 @@
+[
+  {
+    "this": "Object {...}",
+    "{exception}": "number 1",
+    "locals": {
+      "FuncLevel1": "function <large string>",
+      "outerFunc1": "function <large string>",
+      "globalVar": "undefined undefined"
+    }
+  },
+  {
+    "this": "Object {...}",
+    "{exception}": "Error Caught Error",
+    "locals": {
+      "FuncLevel1": "function <large string>",
+      "outerFunc1": "function <large string>",
+      "globalVar": "undefined undefined"
+    }
+  },
+  {
+    "this": {
+      "FuncLevel1": "function <large string>",
+      "outerFunc1": "function <large string>",
+      "globalVar": "Object {...}"
+    },
+    "arguments": {
+      "#__proto__": "Object {...}",

```

Comment:
I wonder why # before __proto__, you would have hard time matching jsrt based baselines with pdm based baseline.